src/queueAdapters/bull.ts

```diff
@@ -1,17 +1,18 @@
-import { Job, Queue } from 'bull'
 import {
+  AdapterOptions,
   JobCleanStatus,
   JobCounts,
   JobStatus,
   QueueAdapter,
 } from '../@types/app'
+import { Job, Queue } from 'bull'
 
 export class BullAdapter implements QueueAdapter {
   public get client() {
     return Promise.resolve(this.queue.client)
   }
 
-  constructor(public queue: Queue) {}
+  constructor(public queue: Queue, public options: AdapterOptions) {}
```

Comment:
Missing implementation on the `BullMQAdapter` as well

Reply:
Yeah, I'm not using the other Adapter, so perhaps someone else can add & test that?

Reply:
It has the same `getJob` implementation, with the same fields.

Reply:
Okay will update I just wont be able to test 🍕

Reply:
Done